src/MissionManager/GeoFenceController.cc

```diff
@@ -495,30 +498,97 @@ bool GeoFenceController::supported(void) const
     return (_managerVehicle->capabilityBits() & MAV_PROTOCOL_CAPABILITY_MISSION_FENCE) && (_managerVehicle->maxProtoVersion() >= 200);
 }
 
-// Hack for PX4
+// Hack for PX4 & APM
```

Comment:
Good point, if I understand the code base right, this is a hack because this code (especially now that it contains an if/else for px4/apm, should rightly live in the apm/px4 firmwareplugins and perhaps be exposed by a Q_PROPERTY in vehicle.h. Is that right?

As for the naming, yes I expect getParamCircularFenceRadius would be more self descriptive. (And a comment as well for what firmware functions this is accessing)